sunpy/__init__.py

```diff
@@ -25,9 +25,10 @@
 from sunpy.util import system_info
 from sunpy.tests import main as self_test
 
-# Sample data
-from sunpy.data.sample import (AIA_171_IMAGE, RHESSI_IMAGE, EIT_195_IMAGE,
-                               RHESSI_EVENT_LIST, CALLISTO_IMAGE)
-
 # Load user configuration
 config = load_config()
+
+# Create shortcuts to sample data
+import sunpy.data.sample
```

Comment:
Having this here means you have to have downloaded the data or you can't import SunPy. So I think this should be removed.